engine/components-api/src/main/java/com/cloud/resource/ResourceManager.java

```diff
@@ -156,6 +156,8 @@ public interface ResourceManager extends ResourceService, Configurable {
 
     List<HostVO> findHostByGuid(long dcId, String guid);
 
+    List<HostVO> findHostByGuidByStatus(long dcId, String guid, Status status);
```

Comment:
can you please rename this findHostByGuidByStatus -> findHostByGuidAndStatus

Reply:
updated